mlflow/metrics/utils/prompts/v1.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List
+
+from mlflow.metrics.base import (
+    EvaluationExample,
+)
+from mlflow.metrics.utils.prompt_template import (
+    PromptTemplate,
+)
+
+# TODO: Update the default_mode and default_parameters to the correct values post experimentation
+default_model = "openai:/gpt4"
+default_parameters = {
+    "temperature": 0.0,
+    "max_tokens": 100,
+    "top_p": 1.0,
+}
+grading_system_prompt_template = PromptTemplate(
+    """
+    Please act as an impartial judge and evaluate the quality of the provided output which
```

Comment:
Can we remove indent?

Reply:
<img width="742" alt="Screen Shot 2023-09-02 at 13 01 31" src="https://github.com/mlflow/mlflow/assets/17039389/92ca92e7-eb04-4cd5-872a-2f5f20a32741">

<img width="742" alt="Screen Shot 2023-09-02 at 13 01 54" src="https://github.com/mlflow/mlflow/assets/17039389/46337176-6df2-4e44-b1fb-815d77e6277d">